### DISCUSSION TITLE
CSLA 5.3 + .NET 5

### DISCUSSION THREAD

**Discussion by uwefms** [MarimerLLC/csla#1834] [Questions] [answered]
I am about to start a new blazor project with .NET 5. Is it possible to use CSLA 5.3 with .NET 5 or should I wait for CSLA 6. If this is not possible I would start with .NET 3.1 and do the shift later.

 regards

    Uwe

**Comment by rockfordlhotka** [comment]
I believe it will work fine.

The only part where you may encounter issues would be with the Blazor specific assemblies, as they are linked to v3.1.

But csla core should work fine.

**Comment by Chicagoan2016** [comment, marked as answer]
@uwefms , @rockfordlhotka  we are also in the same and  looking to start a new project, I think we should stick with v3.1 and upgrade later?

**Reply by rockfordlhotka** [reply]
No doubt that'd be the safest approach, yes.
